core/src/components/item/item.ionic.scss

```
@@ -88,3 +88,22 @@
     box-sizing: border-box;
   }
 }
+
+// Item: Disabled
+// --------------------------------------------------
+
+:host(.item-disabled) {
+  cursor: default;
+  pointer-events: none;
+
+  &::after {
+    @include globals.position-horizontal(0, 0);
+    position: absolute;
+    top: 0;
+    bottom: 0;
```

```suggestion
    @include globals.position(0, 0, 0, 0);

    position: absolute;
```

We can use the `position` mixin here since we are also setting top & bottom. 🙂